src/main/java/techreborn/packets/ServerboundPackets.java

```diff
@@ -46,16 +48,17 @@
 import techreborn.blockentity.storage.item.StorageUnitBaseBlockEntity;
 import techreborn.config.TechRebornConfig;
 import techreborn.init.TRContent;
+import techreborn.packets.serverbound.*;
 
 public class ServerboundPackets {
 
-	public static final Identifier AESU = new Identifier(TechReborn.MOD_ID, "aesu");
-	public static final Identifier AUTO_CRAFTING_LOCK = new Identifier(TechReborn.MOD_ID, "auto_crafting_lock");
-	public static final Identifier ROLLING_MACHINE_LOCK = new Identifier(TechReborn.MOD_ID, "rolling_machine_lock");
-	public static final Identifier STORAGE_UNIT_LOCK = new Identifier(TechReborn.MOD_ID, "storage_unit_lock");
-	public static final Identifier FUSION_CONTROL_SIZE = new Identifier(TechReborn.MOD_ID, "fusion_control_size");
-	public static final Identifier REFUND = new Identifier(TechReborn.MOD_ID, "refund");
-	public static final Identifier CHUNKLOADER = new Identifier(TechReborn.MOD_ID, "chunkloader");
+	//public static final Identifier AESU = new Identifier(TechReborn.MOD_ID, "aesu");
+	//public static final Identifier AUTO_CRAFTING_LOCK = new Identifier(TechReborn.MOD_ID, "auto_crafting_lock");
+	// public static final Identifier ROLLING_MACHINE_LOCK = new Identifier(TechReborn.MOD_ID, "rolling_machine_lock");
+	//public static final Identifier STORAGE_UNIT_LOCK = new Identifier(TechReborn.MOD_ID, "storage_unit_lock");
+	//public static final Identifier FUSION_CONTROL_SIZE = new Identifier(TechReborn.MOD_ID, "fusion_control_size");
+	//public static final Identifier REFUND = new Identifier(TechReborn.MOD_ID, "refund");
+	//public static final Identifier CHUNKLOADER = new Identifier(TechReborn.MOD_ID, "chunkloader");
```

Comment:
These can be removed 👍

Reply:
Wanna keep it to track progress. Will remove as soon as I finish with TR payloads